src/Entity/Location/Location.php

```diff
@@ -54,4 +70,37 @@ public function setAddress(string $address): self
 
         return $this;
     }
+
+    /**
+     * Get activities.
+     *
+     * @return Collection<int, Activity>
+     */
+    public function getActivities(): Collection
+    {
+        return $this->activities;
+    }
+
+    public function addActivity(Activity $activity): self
+    {
+        if (!$this->activities->contains($activity)) {
+            $this->activities[] = $activity;
```

Comment:
Maybe an additional check here that the activity also has this location associated with it?

Reply:
We'll address this in a separate PR